src/lib.rs

```diff
@@ -121,7 +121,6 @@
 //!
 //! The third method [get_dependencies](crate::solver::DependencyProvider::get_dependencies)
 //! aims at retrieving the dependencies of a given package at a given version.
-//! Returns [None] if dependencies are unknown.
```

Comment:
It returns `Result<Dependencies<DP::P, DP::VS, DP::M>, DP::Err>`, not an option.